libres/tests/CMakeLists.txt

```diff
@@ -7,7 +7,9 @@ include(Catch)
 
 add_executable(ert_test_suite analysis/modules/test_ies_enkf_main.cpp
                               analysis/test_enkf_linalg.cpp
-                              res_util/test_matrix.cpp)
+                              res_util/test_matrix.cpp
+							  enkf/enkf_obs_paths_detailed.cpp
```

Comment:
Seems to be tabs inserted here which is mismatched with tabstop of github. Suggest replacing with spaces.